granulate_utils/logging/__init__.py

```diff
@@ -0,0 +1,138 @@
+#
```

Comment:
Would rename folder to `glogger-sdk` and put it in its own file instead of __init__.py (unless you have a good reason to put it here)

Reply:
renamed.